### PR TITLE
llvm: Fall back for missing objcopy

### PR DESCRIPTION
**Pull request by gebart**
Copied from Makefile.include.gnu

### CODE REVIEW

**Comment by gebart** [comment]
This should be a quick and easy review, any takers? @OlegHahm @kaspar030

File: cpu/Makefile.include.llvm [hunk 1]
```diff
@@ -17,7 +17,11 @@ export NM          = $(LLVMPREFIX)nm
 #export LINKER      = $(LLVMPREFIX)ld
 # objcopy does not have a clear substitute in LLVM, use GNU binutils
 #export OBJCOPY     = $(LLVMPREFIX)objcopy
-export OBJCOPY     = $(PREFIX)objcopy
+export OBJCOPY    ?= $(shell command -v $(PREFIX)objcopy gobjcopy objcopy | head -n 1)
```
Comment (kaspar030) [inline cpu/Makefile.include.llvm]: Seems the indent is (and was) off. Fix if you feel like.

Comment (gebart) [inline cpu/Makefile.include.llvm]: The line above is off indent because of the comment prefix.

Comment (kaspar030) [inline cpu/Makefile.include.llvm]: Yup, sorry!

**Pull request merged**